releaseNotes/MicrosoftExtensionsConfigurationAzureAppConfiguration.md

```diff
@@ -1,6 +1,14 @@
 ## Microsoft.Extensions.Configuration.AzureAppConfiguration
 ### [Package (NuGet)](https://www.nuget.org/packages/Microsoft.Extensions.Configuration.AzureAppConfiguration)
 
+### 4.0.0-preview - July 23, 2020
+* Added `IConfigurationRefresher.SetDirty` method  to set the cached value for key-values registered for refresh as dirty. For users that use push notifications from event grid to receive configuration change updates, this allows the cached values to be revalidated on the next call to `RefreshAsync` or `TryRefreshAsync`.
```

Comment:
I will reverse the order. The introduction of the API is not the reason. It's just the result of our support for the push model. How about this?

Added enhanced support for applications that leverage Event Grid integration in App Configuration for configuration refresh with push notifications of configuration changes. The new API below is introduced in `IConfigurationRefresher` interface, which can be called when an application responds to events from Event Grid. It will signal the application to reassess whether configuration should be updated upon the next call to `RefreshAsync()` or `TryRefreshAsync()`.
```cs
void SetDirty(TimeSpan? maxDelay = null)
```

Reply:
BTW, we should add a link to issue https://github.com/Azure/AppConfiguration-DotnetProvider/issues/133.